network/loadbalancers/nginx/variable.tf

```diff
@@ -0,0 +1,11 @@
+variable "version" {
+  default = "1.13.1"
```

Comment:
The nginx version should be an input variable.

Reply:
done